<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameInfoWindow</class>
 <widget class="QDialog" name="FrameInfoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detailed Frame Information</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>8</number>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="lblFrameID">
             <property name="text">
              <string>Frame IDs:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblUniqueID">
             <property name="text">
              <string>Unique IDs</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="listFrameID">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Details:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTreeWidget" name="treeDetails">
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOn</enum>
             </property>
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSave">
             <property name="text">
              <string>Save details to file</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_time">
             <property name="text">
              <string>Interval Histogram</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCustomPlot" name="timeHistogram" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="2,0,2">
       <item>
        <layout class="QGridLayout" name="gridUpper"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Bytes Graph</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLower" rowstretch="0"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>listFrameID</tabstop>
  <tabstop>treeDetails</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>updateDetailsWindow(QListWidgetItem*)</slot>
 </slots>
</ui>
